docs/source/usage_guides/distributed_inference.mdx

```diff
@@ -0,0 +1,107 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Distributed Inference with 🤗 Accelerate
+
+Distributed inference is a common use case, especially with natural language processing (NLP) models. Users often want to
+send a number of different prompts, each to a different GPU, and then get the results back. This also has other cases
+outside of just NLP, however for this tutorial we will focus on just this idea of each GPU receiving a different prompt,
+and then returning the results.
+
+## Common Confusions
```

Comment:
I'm not sure this title is really adapted reading after. This is more an exposition of the problem to solve so maybe call it `## The problem`